components/automate-ui/src/app/components/notifications/notifications.component.html

```diff
@@ -1,9 +1,12 @@
-<div class="notifications">
-  <ng-container *ngFor="let notification of notifications">
-    <chef-notification *ngIf="notification.type !== 'license'"
-      [type]="notification.type"
-      [timeout]="notification.timeout" (dismissed)="handleNotificationDismissal(notification.id)">
-      {{ notification.message }}
-    </chef-notification>
+<div class="notifications"> 
+  <!-- @todo - custom reverse pipe -->
+  <ng-container *ngFor="let notification of notifications.slice().reverse()">
```

Comment:
@susanev - for UX, i reversed the list because I thought it looked better having items added to the bottom of the list, but I don't know if you have newest at the top for a reason.

for devs - If UX keeps the list reversed, I'll create a custom pipe for this.  If not...then theres no pipe to be made.

Reply:
yah latest should be at the top, with others shifting down.